docs/concepts/standard-library/banker.md

```diff
@@ -0,0 +1,20 @@
+---
+id: banker
+---
+
+# Banker
+
+The Banker's main purpose is to handle balance changes of [native coins](./coin.md) within Gno chains. This includes issuance, transfers, and burning of coins. 
+
+The Banker module can be cast into 4 subtypes of bankers that expose different functionalities and safety features within your packages and realms.
+
+[//]: # (The banker module is injected into the GnoVM runtime at execution. )
```

Comment:
Is this link leftover?

Reply:
[b3e5b88](https://github.com/gnolang/gno/pull/1468/commits/b3e5b8851ecff59690055c38ebd0877eadfbe653)